### PR TITLE
data: HarshPanchal18

### PR DESCRIPTION
**Pull request by HarshPanchal18**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6812"></a>

### CODE REVIEW

File: data/HarshPanchal18.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+  "name": "Harsh Panchal",
+  "type": "personal",
+  "bio": "Hi, I'm HARSH!\
+I'm a Software engineer in growth mode, lifelong-learner, and reader.\
+Native Android Programming, Backend, observing and developing tools always excite me.\
+Additionally, I enjoy listening to music, reading books, and acquiring new knowledge.",
```
Comment (HassanTanveer) [inline data/HarshPanchal18.json:7]:
```suggestion
  "bio": "Hi, I'm HARSH!\nI'm a Software engineer in growth mode, lifelong-learner, and reader.\nNative Android Programming, Backend, observing and developing tools always excite me.\nAdditionally, I enjoy listening to music, reading books, and acquiring new knowledge.",
```
Please use `\n` for new line

**Review by HassanTanveer — CHANGES_REQUESTED** [review]

**Review by iamthecloverly — DISMISSED** [review]

File: data/HarshPanchal18.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
+{
+  "name": "Harsh Panchal",
+  "type": "personal",
+  "bio": "Hi, I'm HARSH!\n\tI'm a Software engineer in growth mode, lifelong-learner, and reader.\nNative Android Programming, Backend, observing and developing tools always excite me.\nAdditionally, I enjoy listening to music, reading books, and acquiring new knowledge.",
+
+  "tags": ["Open Source", "DevSecOps", "Python", "Kotlin", "Android", "Jetpack Compose", "Linux"]
```
Comment (kumarsonsoff3) [inline data/HarshPanchal18.json:6]: Please add the missing comma here. 
```suggestion
  "tags": ["Open Source", "DevSecOps", "Python", "Kotlin", "Android", "Jetpack Compose", "Linux"],
```

**Review by kumarsonsoff3 — DISMISSED** [review]

**Review by HassanTanveer — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**